TopMovies/View/TopMoviesVC/SeeAllMoviesCollectionViewCell.swift

```diff
@@ -32,7 +32,12 @@ final class SeeAllMoviesCollectionViewCell: UICollectionViewCell {
     }
   }
   private func setupStyle() {
-    seeMoreImageView.image = UIImage(named: "seeAllMovies") ?? UIImage()
+    if #available(iOS 13.0, *) {
+      seeMoreImageView.image = Asset.Images.seeAllButton.image
+        .withTintColor(Asset.Colors.seeAllButton.color)
+    } else {
+      seeMoreImageView.image = Asset.Images.seeAllButton.image
+    }
```

Comment:
Should this work right here?
```swift
    if #available(iOS 13.0, *) {
       seeMoreImageView.image = Asset.Images.seeAllButton.image
         .withTintColor(Asset.Colors.seeAllButton.color)
     } else {
       seeMoreImageView.image = Asset.Images.seeAllButton.image.withRenderingMode(.alwaysTemplate)
       seeMoreImageView.image.tintColor = Asset.Colors.seeAllButton.color
     }
```

Reply:
The .image doesn't have a tintColor property, the ImageView does, but it doesn't work as expected. Actually I should only have 2 image options for any and dark appearance, but if I don't, there is only a way to use the .withTintColor () method if iOS 13 is installed, or use the default image if not.